src/processor.ts

```diff
@@ -74,6 +85,24 @@ const eventArgsData: eventArgsData = {
     },
 };
 
+export const getCirculatingSupplyProcessRange = (): BlockRangeOption => {
+    const isTestnet = process.env.BITCOIN_NETWORK?.toLowerCase() !== "mainnet";
+    if (isTestnet) {
+        return {};
+    }
+    
+    const nativeCurrency = getNativeCurrency();
+    return {
+        range: {
+            from: nativeCurrency === Token.KINT ? 3250849 : 2959964,
```

Comment:
Starting block heights could be put into separate constants.

Reply:
:heavy_check_mark: commit [9808ce0](https://github.com/interlay/interbtc-squid/pull/124/commits/9808ce0d9f0731d1a7d6e0c661259a667f384165)